packages/ibm-products/src/components/Datagrid/Datagrid/DraggableElement.js

```diff
@@ -99,96 +34,56 @@ const DraggableElement = ({
           `${blockClass}__draggable-handleStyle`
         )}
       >
-        {isSticky ? <Locked size={16} /> : <Draggable size={16} />}
+        {isSticky ? <Locked size={16} /> : <DraggableIcon size={16} />}
       </div>
       {children}
     </>
   );
+
   return (
-    <li
-      className={cx({
-        [`${blockClass}__draggable-handleHolder-isOver`]: isOver && !disabled,
-        [`${blockClass}__draggable-handleHolder-grabbed`]: isGrabbed,
-        [`${blockClass}__draggable-handleHolder-selected`]: selected,
-        [`${blockClass}__draggable-handleHolder--sticky`]: isSticky,
-        [`${blockClass}__draggable-handleHolder`]: !selected,
-      })}
-      ref={ref}
-      aria-selected={isFocused}
-      role="option"
-      tabIndex={isFocused ? 0 : -1}
-      onKeyPress={(e) => {
-        if (e.key === ' ' && e.target === e.currentTarget && !disabled) {
-          const positionText = positionLabel
-            .replace('{index}', index + 1)
-            .replace('{total}', listData.length);
-          const grabAriaText = (
-            isGrabbed ? droppedLabel : grabbedLabel
-          ).replace('{itemName}', ariaLabel);
-          onGrab(grabAriaText + positionText);
-          setIsGrabbed(!isGrabbed);
-          e.preventDefault();
-        }
-      }}
-      onKeyDown={(e) => {
-        if (e.key === 'ArrowUp' || e.key === 'ArrowDown') {
-          onArrowKeyDown(e, isGrabbed, index);
-        }
-      }}
-      onBlur={(e) => {
-        // handle when focus move to inner elements
-        setIsFocusedOnItem(e.currentTarget === e.target);
-      }}
-      onFocus={(e) => {
-        // handle when focus move to li element
-        setIsFocusedOnItem(e.currentTarget === e.target);
-      }}
-    >
-      <span className={`${blockClass}__shared-ui--assistive-text`}>
-        {ariaLabel}
-      </span>
-      {isDragging && !isOver ? (
-        <div
-          ref={preview}
```

Comment:
Preview not implemented in beautiful, needs a little thought as to how to implement. 

Also worth noting that dragged items are not visible outside of the modal.